lessons/codap-transformers2/langs/en-us/pages/codap-build.adoc

```diff
@@ -0,0 +1,84 @@
+= Build Attribute
+++++
+<style>
+body.workbookpage td .autonum::after { content: ')' !important; }
+/* Cram everything to the top instead of distributing space equally */
+#content { display: block !important; }
+</style>
+++++
+
+Make sure you’re logged into the @starter-file{animals} in CODAP. Select the Plugins icon, then choose Transformers.
+
+@vspace{2ex}
+
+== Create, Apply, and Save a Build Attribute Transformer (Step by Step)
+
+[cols="5, 10", frame="none", grid="none", stripes="none"]
+|===
+
+|
+@image{../images/build-ui-annotated.png, 350}
+
+|
+@n Choose Build Attribute from the drop-down menu (Box 1).
+
+@vspace{1ex}
+
+@n Enter `build-is-heavy` as the name (Box 2). What does this name tell you about the Transformer we are creating?
+
+@fitb{}{@ifsoln{It will build a column that states `true` for heavy animals and `false` for lightweight animals.}}
+
+@vspace{1ex}
+
+@n Select the dataset you'd like to transform (Box 3).
+
+@vspace{1ex}
+
+@n Let's name our new attribute `Heavy` (Box 4). What happened to the starred text (left) when you named the attribute?
+
```

Comment:
ok. I finally noticed what you're getting at here because the directions are slightly different than they were on the last page (or maybe because something different happens). Instead of asking a question here, I would encourage you to tell people what you're getting at
"Notice that CODAP replaced the blank in the starred line of text (left) with the attribute name you selected!"